accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/repositories/IHistoryRepository.java

```diff
@@ -31,4 +32,15 @@
 
     List<OPERATION_ENTITY> findAllByAccession(ACCESSION accession);
 
+    List<OPERATION_ENTITY> findAllByMergeInto(ACCESSION accession);
+
+    List<OPERATION_ENTITY> findAllBySplitInto(ACCESSION accession);
+
+    default List<OPERATION_ENTITY> findAllInvolvedIn(ACCESSION accession) {
+        List<OPERATION_ENTITY> allOperations = new ArrayList<>();
+        allOperations.addAll(findAllByAccession(accession));
```

Comment:
I wonder if it is better to have a single query to the database instead of 3

Reply:
I tried to write a single query with Jpa syntax but was not able to make it work. I guess the syntax was wrong. Have updated with correct syntax